src/js/yeldaWebchatInjector.js

```diff
@@ -158,25 +159,37 @@ class YeldaChat {
         return
       }
 
-      const customImage =  settings.data.image && settings.data.image.url
+      const isVoiceFirstUI = settings.data.hasOwnProperty('isVoiceFirstUI') || false
+      const customImage = settings.data.image && settings.data.image.url
       const hasCustomStyle = settings.data.hasOwnProperty('isDefaultStyle') && !settings.data.isDefaultStyle
 
-      if (!hasCustomStyle || !customImage) {
-        this.webChatContainer.appendChild(this.assistantImage)
-        return
-      }
+      /**
+       * when isVoiceFirstUI is true, instead of adding the assistant image open the chat here,
+       * website-chat will take care rendering the voice first UI
+       */
+      if (!data.isDemo && isVoiceFirstUI) {
+        // voiceFirstUI added to the iframeContainer to remove box-shadow css style
+        // other styles can be added for voice first UI based on this class in the future if needed
```

Comment:
To 100% honest I don't fully understand the English here

     /**
       * in isVoiceFirstUI mode
       * - the assistant should be opened directly and will never be closed 
       *    => we call directly openChat and do not add the assistant image
       * - we don't want the box-shadow css style 
       *    => we add voiceFirstUI to iframeContainer
       * - /chat vue render the voice first UI
       *   => nothing more to do here
       */